templates/theme-head-extra.html

```diff
@@ -12,6 +20,8 @@
 
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.css">
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/js/bootstrap.js"></script>
+<script src="http://ajax.aspnetcdn.com/ajax/jquery.validate/1.14.0/jquery.validate.js"></script>
```

Comment:
@mbifulco this line is no good. It works fine under devstack, but staging and production are loaded over HTTPS, so I'm seeing this error:
```
Mixed Content: The page at 'https://courses.gymna.si/login' was loaded over HTTPS, but requested an insecure script 'http://ajax.aspnetcdn.com/ajax/jquery.validate/1.14.0/jquery.validate.js'. This request has been blocked; the content must be served over HTTPS.
```
Does the CDN support HTTPS? If not, is there another CDN serving the plugin or can it just be loaded directly from the server with the other staticfiles?